appintro/src/main/java/com/github/paolorotolo/appintro/internal/viewpager/ViewPagerTransformer.kt

```diff
@@ -56,13 +56,16 @@ internal class ViewPagerTransformer(
             }
             TransformType.FADE -> {
                 if (position <= -1.0f || position >= 1.0f) {
+                    page.translationX = page.width.toFloat()
```

Comment:
This change should not be here.
Can you please rebase?

Reply:
Me not know git😅

Reply:
This is a good starting guide: http://rogerdudler.github.io/git-guide/